compiler/z/codegen/OMRCodeGenerator.hpp

```diff
@@ -1178,6 +1173,12 @@ class OMR_EXTENSIBLE CodeGenerator : public OMR::CodeGenerator
    TR::list<TR::S390ConstantDataSnippet*>  _constantList;
    TR::list<TR::S390ConstantDataSnippet*>  _snippetDataList;
 
+   /**
+    * _processorInfo contains the targetted hardware level for the compilation.
+    * This may be different than the real hardware the JVM is currently running
+    * on, due to user specified options.
+    */
+   TR_S390ProcessorInfo            _processorInfo;
```

Comment:
Awesome to see people documenting related code as part of their changes. Keep it up!